### PR TITLE
[bfdorch] add default TOS value for BFD session

### PR DESCRIPTION
**Pull request by baorliu**
**What I did**
Add default TOS value when create BFD session, default DSCP value is 46, default ECN is 0. The BFD session configuration will override this default value.

**Why I did it**
Higher DSCP value is preferred for BFD packet.

**How I verified it**
Set different TOS value in BFD session configuration, and create BFD session in DUT.
Load the test image with the design change, connect a traffic generator (with BFD protocol support), capture the packet sent out by DUT, check the DSCP and ECN field in the captured packet.

```
cisco@sonic:/var/tmp$ cat bfd.cfg
[
  {
      "BFD_SESSION_TABLE:default:default:192.168.88.2": {
      "tos": "186",
      "multihop": "true",
      "rx_interval": "1000",
      "tx_interval": "1000",
      "local_addr": "192.168.88.1"
  },
  "OP": "SET"
}
]
```
image

APP_DB dump:
```
 cisco@sonic:/var/tmp$ head -n 50 redis0.log 
{
  "BFD_SESSION_TABLE:default:default:192.168.88.2": {
    "expireat": 1677887525.8233933,
    "ttl": -0.001,
    "type": "hash",
    "value": {
      "local_addr": "192.168.88.1",
      "multihop": "true",
      "rx_interval": "1000",
      "tos": "186",
      "tx_interval": "1000"
    }
  },
  ```
  ASIC DB dump:
  ```
    "ASIC_STATE:SAI_OBJECT_TYPE_BFD_SESSION:oid:0x45000000000989": {
    "expireat": 1677887399.4435563,
    "ttl": -0.001,
    "type": "hash",
    "value": {
      "SAI_BFD_SESSION_ATTR_BFD_ENCAPSULATION_TYPE": "SAI_BFD_ENCAPSULATION_TYPE_NONE",
      "SAI_BFD_SESSION_ATTR_DST_IP_ADDRESS": "192.168.88.2",
      "SAI_BFD_SESSION_ATTR_IPHDR_VERSION": "4",
      "SAI_BFD_SESSION_ATTR_LOCAL_DISCRIMINATOR": "6",
      "SAI_BFD_SESSION_ATTR_MIN_RX": "1000000",
      "SAI_BFD_SESSION_ATTR_MIN_TX": "1000000",
      "SAI_BFD_SESSION_ATTR_MULTIHOP": "true",
      "SAI_BFD_SESSION_ATTR_MULTIPLIER": "10",
      "SAI_BFD_SESSION_ATTR_REMOTE_DISCRIMINATOR": "0",
      "SAI_BFD_SESSION_ATTR_SRC_IP_ADDRESS": "192.168.88.1",
      "SAI_BFD_SESSION_ATTR_TOS": "186",
      "SAI_BFD_SESSION_ATTR_TYPE": "SAI_BFD_SESSION_TYPE_ASYNC_ACTIVE",
      "SAI_BFD_SESSION_ATTR_UDP_SRC_PORT": "49157",
      "SAI_BFD_SESSION_ATTR_VIRTUAL_ROUTER": "oid:0x3000000000042"
    }
  },
  ```
  
**Details if related**

### CODE REVIEW

**Comment by baorliu** [comment]
BFD related UT passed. the vstest failure caused by other UT.
```
2023-03-04T03:26:50.4079122Z test_addRemoveBfdSession passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4079402Z test_addRemoveBfdSession_ipv6 passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4079694Z test_addRemoveBfdSession_interface passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4079980Z test_addRemoveBfdSession_txrx_interval passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4080285Z test_addRemoveBfdSession_multiplier passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4080582Z test_addRemoveBfdSession_multihop passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4080893Z test_addRemoveBfdSession_type passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4081203Z test_multipleBfdSessions passed 1 out of the required 1 times. Success!
2023-03-04T03:26:50.4081478Z test_bfd_state_db_clear passed 1 out of the required 1 times. Success!

**Comment by lguohan** [comment]
@prsunny , can yu review this?

File: orchagent/bfdorch.cpp [hunk 1]
```diff
@@ -243,6 +243,7 @@ bool BfdOrch::create_bfd_session(const string& key, const vector<FieldValueTuple
     uint32_t tx_interval = BFD_SESSION_DEFAULT_TX_INTERVAL;
     uint32_t rx_interval = BFD_SESSION_DEFAULT_RX_INTERVAL;
     uint8_t multiplier = BFD_SESSION_DEFAULT_DETECT_MULTIPLIER;
+    uint8_t tos = 184;  // default value 46<<2. higher 6-bit tos 46, lower 2-bit ecn 0
```
Comment (prsunny) [inline orchagent/bfdorch.cpp:246]: please use a # define to declare this

Comment (baorliu) [reply orchagent/bfdorch.cpp]: fixed. thanks.

**Comment by siqbal1986** [comment]
can you please update it to reflect the use of #defile and change the value to 48 as done in https://github.com/sonic-net/sonic-swss/pull/2690

**Comment by prsunny** [comment]
can you please raise a PR to 202012?

**Review by siqbal1986 — APPROVED** [review]
looks good

**Pull request merged**